ztoc/ztoc_marshaller_1x.go

```diff
@@ -0,0 +1,190 @@
+package ztoc
```

Comment:
This file has the modified `flatbufToZtoc` and `ztocToFlatbuffer`. The only change is that I removed all of the references to `build_tool_identifier`.